doc/architecture/database_schemata.md

```diff
@@ -149,3 +144,69 @@ subfamily }|--|| family : ""
 | **is_tree**                      | true                             | Set of Herbaceous/Woody (woody) AND life cycle (perennial)                                        |
 | **nutrition_demand**             | NULL                             | If "Nutritionally poor soil" in `environmental_tolerances` is given `light feeder` should be set. |
 | **preferable_permaculture_zone** | NULL                             |                                                                                                   | -1..6 (-1 should be printed as 00) |
+
+## `Relation`
+
+Many-to-many table to store relations between plants, genus, subfamily and family.
+
+| **_Column name_** | **_Example_**              | **_Description_**                                                             |
+| :---------------- | :------------------------- | :---------------------------------------------------------------------------- |
+| **id**            | 1                          | relation id                                                                   |
+| **from_id**       | 1                          | id of the left side of the relation(id of plant, genus, subfamily or family)  |
+| **from_type**     | genus                      | type can be plant, genus, subfamily or family                                 |
+| **to_id**         | 1                          | id of the right side of the relation(id of plant, genus, subfamily or family) |
+| **to_type**       | family                     | type can be plant, genus, subfamily or family                                 |
+| **relation_type** | dislikes                   | type of the relation can be "likes" and "dislikes"                            |
+| **created_at**    | 2023-02-09 14:06:01.451028 | creation timestamp                                                            |
+| **updated_at**    | 2023-02-09 14:06:01.451028 | update timestamp                                                              |
+
+# Example queries
+
+## Get all plants with their hierarchy information
+
+```sql
+SELECT *
+  FROM plant_detail
+  LEFT JOIN genus
+            ON plant_detail.genus_id = genus.id
+  LEFT JOIN subfamily
+            ON plant_detail.subfamily_id = subfamily.id
+  LEFT JOIN family
+            ON plant_detail.family_id = family.id;
+```
+
+## Insert a relation between a plant with a specific genus and a specific family
+
+```sql
+INSERT INTO relations (from_id, from_type, to_id, to_type, relation_type)
+VALUES (1, 'genus', 156, 'family', 'dislikes');
+```
+
+## Get all plants that dislike a specific family
+
+```sql
+SELECT p.id,
```

Comment:
doesn't find anything for me but shouldn't it (I executed the INSERT... above successfully: INSERT 0 1)?

Reply:
Yes, I should have mentioned why the metatables are empty before the review request. Refer to my [answer](https://github.com/ElektraInitiative/PermaplanT/pull/89#issuecomment-1466148887). Let's discuss it on the next meeting

Reply:
solved, metatables are populated with the insertion of the scraper data. checkout the README before